webapp/src/app/pacman-features/modules/admin/user-management/user-management.component.ts

```diff
@@ -647,7 +642,6 @@ export class UserManagementComponent implements OnInit, AfterViewInit {
       return processedData;
     } catch (error) {
       this.tableErrorMessage = this.errorHandling.handleJavascriptError(error);
```

Comment:
Same here, why we need this.tableErrorMessage?

Reply:
> Same here, why we need this.tableErrorMessage?

It is being used in corresponding html template file to show the error in the UI.